wgpu/src/lib.rs

```diff
@@ -3150,7 +3151,16 @@ impl<'a> RenderPass<'a> {
 
     /// Draws primitives from the active vertex buffer(s).
     ///
-    /// The active vertex buffers can be set with [`RenderPass::set_vertex_buffer`].
+    /// The active vertex buffer(s) can be set with [`RenderPass::set_vertex_buffer`].
+    /// Does not use an Index Buffer. If you need this see [`RenderPass::draw_indexed`]
+    ///
+    /// Panics if vertices Range is not within 0..BufferLen / BufferStride.
```

Comment:
wgpu doesn't panic if you set an error handler.
Vertex buffers are completely optional to drawing and the validation there is more complicated than that

maybe something like that
```suggestion
    /// Fails if the vertex range is outside of the range of the vertex range of any set vertex buffer.
```

Reply:
/// Panics if the vertex range is outside of the range of the vertex range of any set vertex buffer unless [on_uncaptured_error](https://docs.rs/wgpu/0.16.1/wgpu/struct.Device.html#method.on_uncaptured_error) is set then will Fail instead.

Or something like this would be good since in most cases people wont know to set this as I did not even know it existed till now.

Reply:
sounds good!
Scrolling around a bit, I looks like we write "will panic" etc. in a whole bunch of places right now and don't be that super-accurate on "uhm actually you can do on_uncaptured_error", so I guess saying it's paniking is just fine and easier to read